### PR TITLE
Website for tropical meeting 2022-02

### PR DESCRIPTION
**Pull request by YueRen**

### CODE REVIEW

File: meetings/2022-02/location.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: meeting
+title: Location
+meeting: true
+meeting_nr: 202202
+---
+
+The OSCAR Meeting in June 2019 takes place in-person in
```
Comment (fingolfin) [inline meetings/2022-02/location.md]:
```suggestion
The OSCAR Meeting in February 2022 takes place in-person in
```

File: meetings/2022-02/registration.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+layout: meeting
+title: Registration
+meeting: true
+meeting_nr: 202109
```
Comment (fingolfin) [inline meetings/2022-02/registration.md]:
```suggestion
meeting_nr: 202202
```

File: meetings/2022-02/registration.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+Registration is closed now.
```
Comment (fingolfin) [inline meetings/2022-02/registration.md]: Is the plan to ever open it? If not, just delete this page?

**Review by fingolfin — CHANGES_REQUESTED** [review]

File: meetings/2022-02/registration.md (hunk 3 above)
Comment (YueRen) [reply meetings/2022-02/registration.md]: Sorry about that. The plan was to have a smallish closed meeting (in part due to the pandemic). I must have forgotten to delete the registration.md after copying it from the summer school folder. I'll fix it now.

**Comment by YueRen** [comment]
Okay, I deleted registration.md, hope everything is fine now.

**Review by fingolfin — APPROVED** [review]
Looks good to me

**Comment by fingolfin** [comment]
I've merged it but would recommend you double check the online result now

**Pull request merged**